utility/README.md

```diff
@@ -1,679 +1,1309 @@
+   GetMsg() Message        # Payload of the evidence; DoubleSign, TimeoutCert, ClientSideChallenge, etc.
+   GetHeight() Big.Int     # Height at which the evidence was collected and submitted
+   GetAuth() Authenticator # Authentication of evidence, can be a digital signature or certificate
 }
 ```
 
+#### 3.8.3 Autonomous
 
-Autonomous state change operations are performed by the protocol according to specific lifecycle triggers linked to consensus. Examples of these events are BeginBlock and EndBlock which occur at the beginning and ending of each height respectively. Rewarding the block producer, processing QuorumCertificates, and automatic unstaking are all illustrative instances of these autonomous state change commands. 
-
+Autonomous state change operations are performed by the protocol according to specific lifecycle triggers linked to consensus. These operations include rewarding block producers, distributing relay rewards, etc.
 
-## 3.8 Governance Protocol
+### 3.9 Governance Protocol
 
-Though most off-chain governance specification is included in the DAO 1.0 Constitution document, the Utility Module’s Governance Protocol defines how the DAO is able to interact with the on-chain protocol on a technical level. 
+Though most off-chain governance specification is included in [Pocket Network's Constitution](https://github.com/pokt-foundation/governance/blob/master/constitution/constitution.md), the Utility Module’s Governance Protocol defines how the DAO is able to interact with the on-chain protocol on a technical level.
 
-The Access Control List or ACL is the primary mechanism that enables live parameterization of Pocket Network 1.0. Specifically, the ACL maintains the knobs that allow active modification of the behavior of the protocol without any forking. As the name suggests, the ACL also maintains the account(s) permissioned to modify any of the parameters. A value is able to be modified by a ParamChangeMsg from the permissioned owner of that parameter. 
+#### 3.9.1 Parameter Updates
 
+The Access Control List (ACL) is the primary mechanism that enables live parameter configuration. Specifically, the ACL maintains the "feature switches" that allow active modification of the behavior of the protocol without forks. The ACL also maintains the account(s) permissioned to modify said parameters. A value can be modified by a `ParamUpdateMsg` from the permissioned owner of that parameter.
 
-```
-# ParamChange Interface
+```go
 type ParamChangeMsg interface {
-  GetAddress()  Address           # Address of sender account; Must be permissioned on ACL
-  GetParamName() String           # The identifier of the knob
-  GetValue() Big.Int              # The modified value of the parameter
+  GetAddress() Address  # Address of sender & signer; must be permissioned through ACL
+  GetParamName() String # The name of the parameter being updated
+  GetValue() any        # The new value of the parameter being modified
```

Comment:
I would consider also an optional short text field used to reference the reason why the action was taken. Could be a URL (IPFS or Forum come to mind)